generated/apimanagement/microsoft.apimanagement/2017-03-01/types.md

```diff
@@ -42,12 +42,13 @@
 * **properties**: [DiagnosticContractProperties](#diagnosticcontractproperties): Diagnostic Entity Properties
 * **type**: 'Microsoft.ApiManagement/service/apis/diagnostics' (ReadOnly, DeployTimeConstant): The resource type
 
-## Resource Microsoft.ApiManagement/service/apis/diagnostics/loggers@2017-03-01
+## Resource Microsoft.ApiManagement/service/apis/diagnostics/loggers@2017-03-01 (WriteOnly)
 * **Valid Scope(s)**: ResourceGroup
 ### Properties
 * **apiVersion**: '2017-03-01' (ReadOnly, DeployTimeConstant): The resource api version
 * **id**: string (ReadOnly, DeployTimeConstant): The resource id
 * **name**: string (Required, DeployTimeConstant): The resource name
+* **properties**: [LoggerContractProperties](#loggercontractproperties) (ReadOnly): The Logger entity in API Management represents an event sink that you can use to log API Management events. Currently the Logger entity supports logging API Management events to Azure Event Hubs.
```

Comment:
This property appearing is a result of [this change that will sub in the PUT operation's response if no GET is defined](https://github.com/Azure/bicep-types-az/pull/805/files#diff-8c3d58262f34ad4e08ca80feda2fa4b3432558a1922526eef37fd22ed6950860R28).